pkg/ibmvpc/sg_analysis.go

```diff
@@ -11,10 +11,13 @@ import (
 	vpcmodel "github.com/np-guard/vpc-network-config-analyzer/pkg/vpcmodel"
 )
 
+const allCidr = "0.0.0.0/0"
+
 type SGAnalyzer struct {
 	sgResource          *vpc1.SecurityGroup
 	ingressRules        []*SGRule
 	egressRules         []*SGRule
+	rulesAreDefault     bool // rules are the default ones
```

Comment:
the name rulesAreDefault suggest that there are more than one rule, and they are all default.
1. consider renaming
2. also rename areRulesDefault()
3. IsDefault() has basically the same meaning, consider give all the same name

Reply:
The issue here is that there is a default `SG`, and this is not what we are checking here. 
Also,  giving the same name to a field and a method drives it crazy.
Names changed, to avoid confusion I added comments.